manual/source/datarules.rst

```diff
@@ -198,20 +198,22 @@ the prefixes reserved by NeXus.
     reserved prefixes; PDBX_
     reserved prefixes; SAS_
     reserved prefixes; SILX_
-
-============  ==================  ============================================  =============================================================
-prefix        use                 meaning                                       URL
-============  ==================  ============================================  =============================================================
-``BLUESKY_``  attributes          reserved for use by Bluesky project           https://blueskyproject.io
-``DECTRIS_``  attributes, fields  reserved for use by Dectris                   https://www.dectris.com
-``IDF_``      attributes          reserved for use by pulsedTD Muon definition  https://www.isis.stfc.ac.uk/Pages/nexus-definition-v27924.pdf
-``NDAttr``    attributes          reserved for use by EPICS area detector       https://github.com/areaDetector
-``NX``        NXDL class          for the class names used with NeXus groups    https://www.nexusformat.org
-``NX_``       attributes          reserved for use by NeXus                     https://www.nexusformat.org
-``PDBX_``     attributes          reserved for the US protein data bank         https://www.rcsb.org
-``SAS_``      attributes          reserved for use by canSAS                    https://www.cansas.org
-``SILX_``     attributes          reserved for use by silx                      https://www.silx.org
-============  ==================  ============================================  =============================================================
+    reserved prefixes; identifier
```

Comment:
It looks like there are some formatting changes.

IMHO, it would be better to separate out layout/formatting changes from content-modifying changes.  This patch should add the `identifier` row, but not modify the existing rows (if possible).